packages/@aws-cdk/aws-sqs/test/test.sqs.ts

```diff
@@ -154,15 +154,15 @@ export = {
       const stack = new Stack();
 
       // WHEN
-      const imports = sqs.Queue.fromQueueArn(stack, 'Imported', 'arn:aws:sqs:us-east-1:123456789012:queue1');
+      const imports = sqs.Queue.fromQueueArn(stack, 'Imported', 'arn:aws:sqs:us-west-2:123456789012:queue1');
 
       // THEN
 
       // "import" returns an IQueue bound to `Fn::ImportValue`s.
-      test.deepEqual(stack.resolve(imports.queueArn), 'arn:aws:sqs:us-east-1:123456789012:queue1');
+      test.deepEqual(stack.resolve(imports.queueArn), 'arn:aws:sqs:us-west-2:123456789012:queue1');
```

Comment:
Can we make a separate test that explicitly passes a region other than the one that is defined for the stack?

Reply:
Yeah it makes sense to have specific test-case. Added.